MAUI/Kanban-Board/Cards.md

```diff
@@ -21,16 +21,16 @@ The default elements of a card can be customized using the below properties of [
 
 {% highlight C# %}
 
-    new KanbanModel()
-    {
-        ID = 1,
-        Title = "iOS - 1002",
-        ImageURL = "Image1.png",
-        Category = "Open",
-        Description = "Analyze customer requirements",
-        IndicatorFill = Colors.Red;
-        Tags = new string[] { "Incident", "Customer" }
-    });
+new KanbanModel()
+{
+    ID = 1,
+    Title = "iOS - 1002",
+    ImageURL = "Image1.png",
```

Comment:
@EzhilarasanElangovan31 at line 15, refer below, create content and update as notes
![image](https://github.com/user-attachments/assets/4d2ccb67-4fdb-45a1-8eaf-3eb083abe9b8)

Reply:
Added a note